app/views/reports/index.html.erb

```diff
@@ -0,0 +1,556 @@
+                    <td class="text-sm" style="padding: 16px 16px;">1.96%</td>
+                    <td class="text-sm" style="color: red;">-2.3%</td>
+                </tr>
+                <tr style="border-top: 1px solid rgba(0,0,0,0.06);">
+                    <td class="text-sm">Registered % of estimated pop.</td>
+                    <td class="text-sm">8.1%</td>
+                    <td class="text-sm" style="padding: 16px 16px;">6.69%</td>
+                    <td class="text-sm" style="color: red;">-1.4%</td>
+                </tr>
+                <tr style="border-top: 1px solid rgba(0,0,0,0.06);">
+                    <td class="text-sm">Q2-2020 cohort BP control</td>
+                    <td class="text-sm">38%</td>
+                    <td class="text-sm" style="padding: 16px 16px;">39%</td>
+                    <td class="text-sm" style="color: green;">1%</td>
+                </tr>
+                <tr style="border-top: 1px solid rgba(0,0,0,0.06);">
+                    <td class="text-sm">Q2-2020 cohort BP missed visit</td>
+                    <td class="text-sm">27%</td>
+                    <td class="text-sm" style="padding: 16px 16px;">36%</td>
+                    <td class="text-sm" style="color: red;">-9%</td>
+                </tr>
+            </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <!-- Control rate of all registered HTN patients -->
+  <!--div class="md:w-1/2 md:pl-2">
+    <div class="mb-4 p-4 rounded-md bg-white shadow-md md:flex md:flex-col md:justify-between md:h-96">
+      <div class="mb-4">
+        <p class="mb-1 font-roboto font-bold text-lg text-black">
+          Control rate of all registered HTN patients
+        </p>
+        <p class="font-roboto font-normal text-sm text-black md:w-2/3">
+          Proportion of registered HTN patients with BP &lt;140/90 at their most recent visit in the last 3 months
+        </p>
+      </div>
+      <div class="mt-8 md:flex-grow md:flex md:items-center md:justify-between md:mt-0">
+        <div class="mb-6 text-center md:flex-grow md:mb-0">
+          <p class="font-roboto font-bold leading-tight text-6xl text-green-200 md:h-20 md:mb-4">
+            <%= @control_rate.values.last.to_f %>%
+          </p>
+          <p class="font-roboto font-bold text-xl text-black md:text-sm">
+            <%= number_with_delimiter(@controlled_patients.values.last, :delimiter => ",") %> of <%= number_with_delimiter(@registrations.values.last, :delimiter => ",") %> patients
+          </p>
+        </div>
+        <div class="mb-4 md:flex md:flex-col md:items-center md:flex-grow">
+          <div class="relative ml-auto mr-auto h-20 w-1/2 md:h-20 md:w-2/3 md:mb-6">
+            <canvas id="controlRateOfAllRegisteredPatients"></canvas>
+          </div>
+          <p class="font-roboto text-center text-sm md:text-sm md:font-medium md:mt-1">
+            Last 12 months
+          </p>
+        </div>
+      </div>
+    </div>
+  </div-->
```

Comment:
Can you remove this commented code if we don't need this section anymore?

Reply:
Sorry about that, changing now @rsanheim